Directory.Build.props

```diff
@@ -12,7 +12,7 @@
     <EnableSourceControlManagerQueries>$(EnableSourceLink)</EnableSourceControlManagerQueries>
     <!--This will target the latest patch release of the runtime released with the current SDK.  -->
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
-    <SdkPackageVersion>3.1.11</SdkPackageVersion>
+    <SdkPackageVersion>5.0.2</SdkPackageVersion>
```

Comment:
>SdkPackageVersion [](start = 5, length = 17)

I would split this variable into two variables:
`<RuntimePackageVersion>5.0.0</RuntimePackageVersion>`
and 
`<AspPackageVersion>5.0.2</AspPackageVersion>`

Reply:
This should be ok, each variable will be split into a separate PR by dependabot.

Reply:
Sounds good! Reverted the change to split the versions as it will be taken care by dependabot.

Reply:
Lol sorry, keep what you had (RuntimePackageVersion & AspNetPackageVersion)! Without the variables we will get multiple PRs from dependabot for packages that should probably version together

Reply:
Sorry to  misunderstand, will revert the reverted :)